prow/flagutil/github.go

```diff
@@ -324,11 +316,19 @@ func (o *GitHubOptions) GitHubClient(dryRun bool) (github.Client, error) {
 }
 
 // GitHubClientWithAccessToken creates a GitHub client from an access token.
-func (o *GitHubOptions) GitHubClientWithAccessToken(token string) github.Client {
+func (o *GitHubOptions) GitHubClientWithAccessToken(token string, usePAT bool) (github.Client, error) {
 	options := o.baseClientOptions()
 	options.GetToken = func() []byte { return []byte(token) }
+	if o.AppPrivateKeyPath != "" {
```

Comment:
Just for my understanding, all of the changes in here and in `prow/github/client.go` are to get the client to use token authorizarion despite `o.baseClientOptions` copying the AppID which makes it use apps auth?

Wouldn't it be much easier to just set `options.AppID = ""`?

Reply:
your understanding is correct.

setting `AppId` as "" also works. I was originally concerned that this might break something else, but after looking through everything I am confident there won't be any unintended side effects here. I have made the update.